install/installer/pkg/components/openfga/deployment.go

```diff
@@ -28,75 +28,10 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil, nil
 	}
 
-	containers := []corev1.Container{
-		{
```

Comment:
The definition is moved below. If there's config for CloudSQL, we configure the sidecar first and based on that we inject extra envs to the openfga application.